src/openforms/registrations/contrib/objects_api/api/serializers.py

```diff
@@ -2,6 +2,18 @@
 
 from rest_framework import serializers
 
+from openforms.api.fields import PrimaryKeyRelatedAsChoicesField
+
+from ..models import ObjectsAPIGroupConfig
+
+
+class ObjectsAPIGroupInputSerializer(serializers.Serializer):
+    objects_api_group = PrimaryKeyRelatedAsChoicesField(
+        queryset=ObjectsAPIGroupConfig.objects.exclude(objecttypes_service=None),
```

Comment:
will this exclude cause problems with the legacy objects API configuration? A group selection is required, but the automatically migrated groups will not be shown :thinking:

Reply:
This serializer is only used to fetch the available objecttypes/objecttype versions, and is only called when a valid group is being selected, so this should be fine?

Reply:
yeah I guess it depends on the queryset being specified correctly for the options serializer - I was a little bit lost on what exactly this serializer was used for but it became clear later :+1: